moveit_core/planning_scene/include/moveit/planning_scene/planning_scene.h

```diff
@@ -77,11 +77,44 @@ typedef std::function<bool(const moveit::core::RobotState&, bool)> StateFeasibil
     whether the check should be verbose or not. */
 using MotionFeasibilityFn = std::function<bool(const moveit::core::RobotState&, const moveit::core::RobotState&, bool)>;
 
+/** \brief A map from object names (e.g., attached bodies, collision objects) to their types */
+using ObjectTypeMap = std::map<std::string, object_recognition_msgs::msg::ObjectType>;
+
 /** \brief A map from object names (e.g., attached bodies, collision objects) to their colors */
 using ObjectColorMap = std::map<std::string, std_msgs::msg::ColorRGBA>;
```

Comment:
I just changed the order of these declarations so `ObjectColorMap` is closer to the color-related stuff. No functional change here.